src/openms/source/KERNEL/MSSpectrum.cpp

```diff
@@ -699,4 +699,17 @@ namespace OpenMS
   bool MSSpectrum::RTLess::operator()(const MSSpectrum &a, const MSSpectrum &b) const {
     return a.getRT() < b.getRT();
   }
+
+  // static
+  bool MSSpectrum::containsIMData() const
```

Comment:
maybe better to return bool and type?

Reply:
what type?

Reply:
I thought type of ion mobility that is stored... maybe some CVTerm ... but not urgent

Reply:
I'm not sure this is well defined. Will omit that for now